Automattic/wp-calypso
# Wires up highlighting of products

#### Changes proposed in this Pull Request

* Declares items to be highlighted and wires up the badge and ProductCard props.

#### Screenshots
<img width="527" alt="Screen Shot 2020-08-17 at 3 59 59 PM" src="https://user-images.githubusercontent.com/1760168/90439851-3e239880-e0a4-11ea-92a1-0e53dc78b4e0.png">

#### Testing instructions



* Enable A/B test for offer reset.
* Visit plans page on a site without Jetpack Security.
* Verify Jetpack Security matches screenshot above.
* Verify treatment does not appear after clicking on product.

**Notes**
- The way this is wired allows for a future hook to get what products have this treatment from state.
- Currently purchasing is not wired up so it's not possible to test.

Fixes 1169247016322522-as-1188453012691999.
